library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -215,13 +215,19 @@ def DoneProvide(error, reason, name)
         return "C"
       end
 
-      # INVALID
-      if error == 3
+      message =
+        case error
+        when 2
```

Comment:
It would be probably a good idea to add a comment like "The 1 case is handled in the MediaChange callback" comment here.